src/components/Quote.js

```diff
@@ -0,0 +1,15 @@
+import './Home.css';
+
+const Quote = () => (
+  <>
+    <h3 className="header">Quote</h3>
+    <p className="math-quote">
+      As far as the laws of mathematics refer to reality, they are not certain,
+      and as far as they are certain,
+      they do not refer to reality.
+      <strong> ~ Albert Einstein</strong>
+    </p>
```

Comment:
- [ Optional ] I think it would be nice if the quotes in the `Quote` component were generated dynamically from an `API`, I've looked at a previous milestone of this project and noticed that at this stage the quotes are supposed to be generated dynamically using the external API: [API Ninjas](https://api-ninjas.com/api/quotes). Please consider adding this functionality to your application.

## Project requirements
![Screenshot from 2023-09-26 17-19-19](https://github.com/yayaismaail/math-magicians/assets/16392046/b6988e79-64df-49f1-a75d-a8ef978265a1)

Reply:
Thank you very much for your feedback.

Reply:
@Felix45, I have added. Once again, accept my deepest thanks